pkg/edgeview/src/basics.go

```diff
@@ -0,0 +1,451 @@
+// Copyright (c) 2021 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"bytes"
+	"crypto/sha256"
+	"fmt"
+	"net"
+	"os"
+	"os/exec"
+	"strings"
+	"time"
+
+	"golang.org/x/crypto/ssh"
```

Comment:
Cannot find it in go.mod

Reply:
it has "golang.org/x/cryptio", i guess that covers this ssh